tests/screenshot/apps/components/Alert.js

```diff
@@ -55,10 +106,39 @@ const overlayColorTests = [
 				</div>
 			</Scroller>
 		</div>
+	</Alert>,
+	// QWTC-2603
+	<Alert open title="With different types of Components">
+		<AlertImage
+			src={img}
+			type="icon"
+		/>
+		<Button size="small">Yes</Button>
+		<Button size="small">No</Button>
+		<div>
+			<div>This is progressbar</div>
+			<ProgressBar progress={0.5} />
+		</div>
+		<div>
+			<CheckboxItem>This is CheckboxItem</CheckboxItem>
```

Comment:
There is no unfocused checked Item case. so I added.
```suggestion
			<CheckboxItem>This is CheckboxItem</CheckboxItem>
			<CheckboxItem selected>This is Selected CheckboxItem</CheckboxItem>
```